stuff/task-x/task-logic/package.json

```diff
@@ -4,12 +4,14 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha tests"
+    "test": "nyc mocha tests"
```

Comment:
@manuelbarzi Mocha use by defould the name of `test` for execute your tests,  if you had called `./tests/**.js` like `./test/**.js` you could have executed `mocha` like this -->
```json
    "test": "mocha",
    "coverage": "nyc mocha",
    "covExport": "nyc --reporter=html --reporter=text mocha",
```

Reply:
UPDATE it to the following, using 'tests', not necessary to rename any folder. this also helps understanding how to run mocha and nyc in a different folder from default one (test). just UPDATE it to:

"test": "mocha tests",   
 "coverage": "nyc mocha tests",
 "coverage-report": "nyc --reporter=html --reporter=text mocha"